src/hotspot_protocol/longfi.proto

```diff
@@ -44,14 +44,17 @@ message LongFiRxPacket {
     uint32 oui = 5;
     // Device ID
     uint32 device_id = 6;
-    // Fingerprint 
+    // Fingerprint
     uint32 fingerprint = 7;
-    // Sequence 
+    // Sequence
     uint32 sequence = 9;
     // Spreading to be used
     LongFiSpreading spreading = 10;
     // the fully reassembled payload
     bytes payload = 11;
+    // De-golayed datagram id and flag bits.
+    // NOTE: only the lowest 12 bits are valid.
+    uint32 tag_bits = 12;
```

Comment:
@Vagabond We should not have to update miner/router because of this right?

Reply:
Additional fields would be ignored if router has a proto without that last one

Reply:
I also need to use this in the router, so....